services/src/main/java/org/apache/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -69,6 +70,7 @@
     name = "validator",
     description = "Validates that a given Druid JSON object is correctly formatted"
 )
+@SuppressForbidden(reason = "System#out")
```

Comment:
Place this annotation on the field like below not work, so I have to annotate the whole class.
```java
@SuppressForbidden(reason = "System#out")
private Writer logWriter = new PrintWriter(new OutputStreamWriter(System.out, StandardCharsets.UTF_8));
```